### DISCUSSION TITLE
Issues with loading local images.

### DISCUSSION THREAD

**Discussion by Puneet56** [nandorojo/solito#184] [Q&A] [answered]
Hi all,
I ma facing an issue whenever I try loading an image using require(../..) syntax I face 
TypeError: unsupported file type: undefined (file: undefined)  error.

The image loads on mobile but not on web. Tried setting disableStaticImage to true in next.config.js but still it does not work. Does anyone else faced the same issue and how did you resolved it?

**Comment by ydrea** [comment]
+1

**Comment by Puneet56** [comment, marked as answer]
[UPDATE]. Removed .babelrc.json from next.js project so that SWC is used and issue is fixed.
